docs/developers/packaging.md

```diff
@@ -2,7 +2,7 @@
 
 ## Option 1 (recommended): Use the `fn` cli tool
 
-We recommend using the [fn cli tool](../fn/README.md) which will handle all of this for you. But if you'd like to dig in
+We recommend using the [fn cli tool](../../../cli/README.md) which will handle all of this for you. But if you'd like to dig in
```

Comment:
Shouldn't we use relative paths? Why not just the github link?

Reply:
The CLI installation is documented in two places - (1) Fn/Readme.md and (2) cli/Readme.md . If we use (1) relative path is fine. If we use (2) we should specify the absolute URL. I have updated the file to point to (1).